engines/pytorch/pytorch-engine/src/test/java/ai/djl/pytorch/integration/ManagerAttachmentTest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ * with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+ * OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package ai.djl.pytorch.integration;
+
+import ai.djl.ndarray.NDArray;
+import ai.djl.ndarray.NDManager;
+import ai.djl.ndarray.index.NDIndex;
+import ai.djl.ndarray.types.Shape;
+import ai.djl.pytorch.engine.PtNDArray;
+import ai.djl.pytorch.engine.PtNDManager;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class ManagerAttachmentTest {
```

Comment:
Instead of creating these tests for both MXNet engine and PyTorch engine, can you move it into the integration module? That is where we put the tests representing behavior shared for the entire engine spec. Engines that don't fully implement the spec can return an UnsupportedOperationException and it will not be considered failing the test